### DISCUSSION TITLE
How do i issue a query that refers to multiple database tables in a query?

### DISCUSSION THREAD

**Discussion by vanderdoes** [pawelsalawa/sqlitestudio#4443] [Q&A] [answered]
On the web page it states:

Multiple databases seamlessly
You can execute SQL statements that refer to multiple databases within single query, thanks to transparent database attaching mechanism built into the SQL Editor of SQLiteStudio.

I tried to query in the discussions but may not be using correct keywords to understand how to do this.  Any insight?

While in sqllite one uses attach, which I think sqlitestudio accepts I can't seem to get it right to work properly.  Any examples would be appreciated.

Thanks,
Jeff

**Comment by pawelsalawa** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/481846/167215190-0f09bd2c-91a9-43b5-9e50-3008cdfec0ac.png)

**Comment by vanderdoes** [comment]
Awesome.  Thanks!

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
